indico/modules/events/editing/client/js/management/editable_type/file_types/FileTypeManager.tsx

```diff
@@ -45,6 +59,25 @@ function fileTypesReducer(state, action) {
   }
 }
 
+function getFileTypeURLs(endpoint) {
```

Comment:
When we discussed this initially, we talked about passing these URL builders to the component as props. Also because the `type` is not a thing in paper peer reviewing, so this part of the URL arg could be handled outside when passing the url builder function promps, e.g. something like this for editing:

```js
getAllURL={params => editingFileTypesURL({type: editableType, ...params})}
```

and this for paper peer reviewing:

```js
getAllURL={paperFileTypesURL}
```

Reply:
Done!